profiles/kentik_snmp/raritan/raritan-pdu.yml

```diff
@@ -0,0 +1,39 @@
+#https://mibs.observium.org/mib/PDU2-MIB/
+---
+extends:
+  - system-mib.yml
+
+provider: kentik-pdu
+
+sysobjectid:
+  - 1.3.6.1.4.1.13742.6    # Raritan PX2 PDU2
+
+metrics:
+  - MIB: RARITAN-PX2-PDU2-MIB
+    table:
+      OID: 1.3.6.1.4.1.13742.6.3.5.3
+      name: outletConfigurationTable
+    symbols:
+      - OID: 1.3.6.1.4.1.13742.6.3.5.3.1.3
+        name: outletName
+metric_tags:
+    column:
+      OID: 1.3.6.1.4.1.13742.6.3.5.3.1.6
+      name: outletRatedVoltage
+    column:
+      OID: 1.3.6.1.4.1.13742.6.3.5.3.1.7
+      name: outletRatedCurrent
```

Comment:
the indentation on this section is off; should be 4 spaces in to align with the `MIB`, `table`, and `symbols` attributes for the tabular OID